foundations/src/sentry/metrics.rs

```diff
@@ -0,0 +1,11 @@
+//! Sentry event related metrics.
+
+use super::Level;
+use crate::telemetry::metrics::Counter;
+
+/// Sentry metrics.
+#[crate::telemetry::metrics::metrics(crate_path = "crate", unprefixed)]
+pub mod sentry_events {
```

Comment:
what do you think about calling this `mod sentry` instead and have the metric be `events_total`?

that makes the names for other metrics in the future more sensible, e.g. `sentry_requests_failed` for the POST requests to the sentry API.

Reply:
Fixed! Makes a lot of sense to me :)